packages/next/client/use-intersection.tsx

```diff
@@ -37,9 +40,11 @@ export function useIntersection<T extends Element>({
   )
 
   useEffect(() => {
+    let idleCallback: any
```

Comment:
Please don't use `any` types

Reply:
@timer `RequestIdleCallbackHandle` is `any` in `request-idle-callback` so I'm not sure what to use here. Should I change both?

Reply:
Aah.. Let's export the RequestIdleCallbackHandle type? 

Actually, we can probably rely on inference if we move the return statement into the if clauses.

Reply:
Good idea, went with that.